cmd/root.go

```diff
@@ -119,7 +119,13 @@ var RootCmd = &cobra.Command{
 func Execute() {
 	if err := RootCmd.Execute(); err != nil {
 		code := -1
-		var logger logrus.FieldLogger = logrus.StandardLogger()
+
+		var logger logrus.FieldLogger = &logrus.Logger{
+			Out:       os.Stderr,
+			Formatter: new(logrus.TextFormatter),
+			Hooks:     make(logrus.LevelHooks),
+			Level:     logrus.InfoLevel,
+		}
```

Comment:
This is because the one from `StandardLogger()` has `ExitFunc: os.Exit` which was previously sidestepping the code below?

Reply:
No, it is because even now configuring the logger is actually changing the `StandardLogger` so the `StandardLogger` here will be the Loki one (for example). But logging to it without waiting for it to push will do nothing. 

Arguably this error should be logged in both loggers IMO, but given that, for it to be "succesfully" logged with anything that won't just write it directly the whole `cmd` package will need to be rewritten (so it waits for a context for example.